tools/train.py

```diff
@@ -22,6 +22,8 @@ def parse_args():
     parser = argparse.ArgumentParser(description='Train a detector.')
     parser.add_argument('config', help='Train config file path.')
     parser.add_argument('--work-dir', help='The dir to save logs and models.')
+    parser.add_argument(
+        '--load-from', help='The checkpoint file to load from.')
```

Comment:
This is typically set in configs. In most of the cases, using it directly means something wrong :)